docs/_toc.yml

```diff
@@ -84,6 +84,14 @@ subtrees:
       - file: plugins/debug_plugins
       - file: plugins/npe2_migration_guide
       - file: plugins/npe1
+      - file: plugins/testing_workshop_docs/index
```

Comment:
I downloaded the build artifact here and the TOC is a bit wild. The `index` page I expected is nowhere to be found and the `subtrees` directive doesn't seem to have worked. Do we need a `- titlesonly` like below in the `community` section? Paging @melissawm...
![image](https://user-images.githubusercontent.com/17995243/203438547-ebdeba05-6b42-4a08-94e1-d5300b0421a9.png)

Reply:
> I downloaded the build artifact here and the TOC is a bit wild. The `index` page I expected is nowhere to be found and the `subtrees` directive doesn't seem to have worked. Do we need a `- titlesonly` like below in the `community` section? Paging @melissawm... ![image](https://user-images.githubusercontent.com/17995243/203438547-ebdeba05-6b42-4a08-94e1-d5300b0421a9.png)

This is happening because the index.md file (and I think some other files in this tree) have no title. The JupyterBook structure is very strict about titles in documents, so it will complain about this when building. 

I will do a full build and review but this should be easy to fix :)

Reply:
Confirming: the ToC looks good, just adding the titles to the documents sould fix this.

Reply:
Ahhhh of course @melissawm thanks for looking into this 🙏

Reply:
I will be happy to change the TOC, I just need someone to show me how. 
Thanks

Reply:
I took the introductory info out of each article and put it in index.md and I made sure each article has a title.  Now I just need to change the TOC.

Reply:
From Melissa: "Confirming: the ToC looks good, just adding the titles to the documents should fix this."
I have added titles to all the documents, renamed them, and just generally cleaned this up.